Automattic/jetpack
# Protect: Introduce accordion components (`Accordion` and `AccordionItem`)

Introduce two new components: `Accordion` and `AccordionItem`.

Both need to be used together.

#### Changes proposed in this Pull Request:

* Introduce two new components: `Accordion` and `AccordionItem`.

#### Jetpack product discussion


p1HpG7-fBP-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start Storybook
* Navigate to `Protect/Components/Accordion`
* You should navigate on it using mouse and keyboard (only Tab).

#### Demo

https://user-images.githubusercontent.com/1663717/163909393-b38fd23e-6fef-4f33-b059-a266ec25e571.mp4
